Resources/doc/features/headers.rst

```diff
@@ -34,7 +34,7 @@ This is an example configuration. For more, see the
                         host: ^login.example.com$
                     headers:
                         cache_control: { public: false, max_age: 0, s_maxage: 0 }
-                        last_modified: "-1 hour"
+                        etag: true
```

Comment:
i would prefer to keep the example as is if its ok to you. its just an example anyways.